addons/hr_timesheet/views/project_portal_templates.xml

```diff
@@ -48,11 +48,15 @@
             <attribute name="t-value">9</attribute>
         </xpath>
         <xpath expr="//thead/tr/th[@name='project_portal_milestones']" position="after">
-            <th t-if="is_uom_day" class="text-end">Days Spent</th>
-            <th t-else="" class="text-end">Hours Spent</th>
+            <t t-if="show_allocated_time">
+                <th t-if="is_uom_day" class="text-end">Days Spent</th>
+                <th t-else="" class="text-end">Hours Spent</th>
+            </t>
         </xpath>
         <xpath expr="//tbody/t/tr/td[@name='project_portal_milestones']" position="after">
+            <t t-if="show_allocated_time" >
             <td class="text-end">
+                <t t-if="task.allow_timesheets">
```

Comment:
```suggestion
            <t t-if="show_allocated_time and task.allow_timesheets" >
                <td class="text-end">
```
- Adjust indentation
- If you split the t-if like you did, you end up with an empty td. Is that intended?

Reply:
Hello @auon-odoo 
If i will merge both condition then it will effect the task portal because there is no empty space so status and stage shift right side. Here allow timesheet is use for show the time for particular task.
you can see the example : [ **https://drive.google.com/file/d/1ju2jySwDouveTbZnKLP9nrQMpXHMrQEO/view?usp=sharing** ]

Thank you:blush: